### ISSUE TITLE
Translated flux grids break BE, when in "Page" module trying to look at translated page version

### ISSUE THREAD

**Issue by InvisibleKind** [FluidTYPO3/flux#963]
Tried to be specific in a title, but the issue is really kinda special.

So, here is more detailed description:
I have a grid CE, made with fluidcontent. Nothing special:
```html
<html xmlns="http://www.w3.org/1999/xhtml"
     xmlns:flux="http://typo3.org/ns/FluidTYPO3/Flux/ViewHelpers"
     xmlns:f="http://typo3.org/ns/TYPO3/Fluid/ViewHelpers">

	<f:layout name="Content" />

	<f:section name="Configuration">
		<flux:form id="grid_50_50">
			<flux:form.option.group value="Custom Grids" />
			<flux:grid>
				<flux:grid.row>
					<flux:grid.column name="content.left" style="width: 50%;" />
					<flux:grid.column name="content.right" style="width: 50%;" />
				</flux:grid.row>
			</flux:grid>
		</flux:form>
	</f:section>

	<f:section name="Main">
		<div class="grid clearfix">
			<div class="grid__column grid__column_left">
				<flux:content.render area="content.left" />
			</div>

			<div class="grid__column grid__column_right">
				<flux:content.render area="content.right" />
			</div>
		</div>
	</f:section>
</html>
```

TYPO3 7.4, 2 languages: EN (default) and DE.
For English version I have a grid with contents inside - all is okay. But as soon as I click "Translate default content elements", I'm getting a PHP error:
```
#1: PHP Warning: Invalid argument supplied for foreach() in /path/typo3_src/typo3/sysext/backend/Classes/Controller/PageLayoutController.php line 1363
```

The error is triggered by `TYPO3\CMS\Backend\Controller\PageLayoutController::isColumnEmpty(0, 0)` at line: 
```php
foreach ($this->contentElementCache[$languageId][$colPos] as $uid => $row) {
```

Playing around with debugger I didn't find much, but only noticed, that `$this->contentElementCache` never contains index with `$colPos=0` when I'm on page with grid, but on other pages it is there.

Another observation: when grid is already translated and throws an error, changing view-mode of "Page" module to "Columns->English" shows English version of page with grid normally, but as soon as I change language to "German", error is there again.

Rolling back to Flux to 7.2.1 solves an issue and no error appears.
7.2.2 and 7.2.3 bring the error back.

**Comment by mariodrengner** [comment]
+1

**Comment by grimmcreative** [comment]
Same issue in 6.2.15 all latest TER

**Comment by DenisMir** [comment]
+1

**Comment by K3CK** [comment]
+1

**Comment by NamelessCoder** [comment]
Fixed with TYPO3 7.6.5 (at time of writing this still unreleased). There is no publicly accessible way to fill the cache before calling the language button function so we have to wait for this version to be released.
